python/cugraph/cugraph/experimental/compat/nx/algorithms/link_analysis/pagerank_alg.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph
+import cugraph.utilities
+import cudf
+import numpy as np
+
+
+def cudify(d):
+    """
+    Converts a dictionary which is what networkx sends to a
```

Comment:
Does `networkx` only returns dictionary . probably reword it to something like `convert networkx dictionary results to a cudf.Dataframe`. cuGraph doesn't require cudf.DataFrame for all inputs parameters. Some of them can be lists